parquet-hadoop/src/test/java/org/apache/parquet/format/converter/TestParquetMetadataConverter.java

```diff
@@ -990,6 +990,30 @@ private void testUseStatsWithSignedSortOrder(StatsHelper helper) {
     }
   }
 
+  @Test
+  public void testFloat16Stats() {
+    BinaryStatistics bStats = new BinaryStatistics();
```

Comment:
I think we may need to define an independent `Float16Statistics` class since there's some special handling required for floating-point min/max values. For instance: https://github.com/apache/parquet-mr/blob/45a1ae2d3b4078d3b9e83919cacb03050c1b8c88/parquet-column/src/main/java/org/apache/parquet/column/statistics/Statistics.java#L83-L107

The situation is a bit unusual, since `Statistics` classes are typically exclusive to physical types. However, I believe [`createStats`](https://github.com/apache/parquet-mr/blob/45a1ae2d3b4078d3b9e83919cacb03050c1b8c88/parquet-column/src/main/java/org/apache/parquet/column/statistics/Statistics.java#L194) is capable of constructing one based on the logical type without having to change the existing API. Ideally, `Float16Statistics` would simply extend `BinaryStatistics` (assuming there aren't any roadblocks to doing that)

Reply:
+1

Reply:
Added a **Float16Statistics** and **Float16Builder** in **Statistics** , please check if it makes sense